client/homebrew/pages/basePages/listPage/listPage.less

```diff
@@ -125,4 +125,73 @@
 
 
 	}
+	.tags-container {
```

Comment:
In this element, there are some properties that are not valid, and some which add nothing.

- font-family is already inherited, so can go
- position:sticky does not really happen, since the page is not scrolling, only the brewlist does. This applies to all nav elements
- if position is not set, top and left can go
- a width of 100% is the default value of the property(can't find the doc that talks about this but any block-level element(flex included)'s default width is auto, which means the element will take up the available width in its containing block.
- text-align:center is not necessary, since there is no text outside of a span in this element
- z-index:1 is also unnecessary, the default works just fine.

Reply:
I think a lot of these are remnants of the original tag container, which was styled quite differently.
Applied in the latest commit.